src/Model/Domain/Tracking/UpdateClickTrackingResponse.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Copyright (C) 2013 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Model\Domain\Tracking;
+
+use Mailgun\Model\ApiResponse;
+
+/**
+ * @author Blake Hancock <blake@osim.digital>
+ */
+final class UpdateClickTrackingResponse implements ApiResponse
+{
+    private $click;
+    private $message;
+
+    public static function create(array $data): self
+    {
+        $model = new self();
+        if (isset($data['click'])) {
+            $model->click = ClickTrackingItem::create($data['click']);
+        }
+
+        $model->message = $data['message'] ?? null;
```

Comment:
I cannot see any reference to `message` next to `click` when checking the documentation.
Can you please tell me where I can find it?

Thanks,

Reply:
It appears to be vestigial remnants from referencing the way other calls of a similar nature were set up. I have removed them.

Reply:
Thanks for the update.

It looks like now the PHPUnit tests are failing (please check Travis CI jobs) due to there's a discrepancy between the calls that we provide and the expected content as part of the tests.

If you can fix it, we can proceed.

Thanks,